documentation/source/directives.rst

```diff
@@ -537,3 +537,11 @@ Config Values
    documentation should be placed immediately after the brief and detailed description
    or at the end, after the returns, remarks and warnings section. Default value and
    also the legacy behavior is False.
+
+.. confval:: breathe_separate_member_pages
+
+   True or False setting to control if the input XML generated by Doxygen had the
+   Doxygen SEPARATE_MEMBER_PAGES option set to YES or NO. The Doxygen option defaults
+   to NO which generates a XML that allows Breathe to resolve all references. When set
```

Comment:
```suggestion
   to NO which generates XML that allows Breathe to resolve all references. When set
```

I feel this is nicer to read, though I might be overly pedantic with this.

(May require re-wrapping of text.)

Reply:
Done